### PR TITLE
social embed added

### PR DESCRIPTION
**Pull request by YashKumarVerma**
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is not applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a live link or preview screenshot
- [x] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [x] Included a description of my change below

# Things done in this Pull Request

- added a new blog entry
- added twitter, facebook and google plus
- hover hang animation on contributor image

Demo:
https://yashkumarverma.github.io/gci17.fossasia.org/

### CODE REVIEW

**Comment by akshatnitd** [comment]
Please share the demo link

**Comment by YashKumarVerma** [comment]
@akshatnitd done 😄

**Comment by YashKumarVerma** [comment]
@mariobehling

**Review by ParthS007 — APPROVED** [review]
LGTM ! Good work

**Comment by YashKumarVerma** [comment]
Can we have this merged please

File: index.html [hunk 1]
```diff
@@ -223,6 +223,38 @@ <h4 class="uppercase mb16">Community Connect</h4>
...
+        </div>
+        <div class="row">
+            <div class="col-sm-4">
+                <div class="feature feature-1 boxed">
+                    <a class="twitter-timeline" data-width="400" data-height="400" data-link-color="#E81C4F" href="//twitter.com/fossasia">Tweets by fossasia</a>
+                </div>
+            </div>
+            <div class="col-sm-4">
+                <div class="feature feature-1 boxed">                 
+					<div class="g-page" data-href="//plus.google.com/108920596016838318216" data-rel="publisher"></div>
```
Comment (anshumanv) [inline index.html]: Indentation

**Review by anshumanv — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (YashKumarVerma) [reply index.html]: @anshumanv sir, but I think it's indebted. Can you please help me with the changes to make?

Comment (YashKumarVerma) [reply index.html]: Sir can you please help me  with the changes  to make?

**Comment by YashKumarVerma** [comment]
@anshumanv Please review the changes
😄

**Review by anshumanv — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (anshumanv) [reply index.html]: I was offline, so couldn't respond

**Comment by YashKumarVerma** [comment]
@thank you @anshumanv  sir :clinking_glasses:

File: index.html [hunk 2]
```diff
@@ -223,6 +223,38 @@ <h4 class="uppercase mb16">Community Connect</h4>
 	</div>
 </section>
 
+<!-- Social Embed -->
+<section class="image-bg overlay parallax">
+    <div class="background-image-holder">
+        <img alt="Background Image" class="background-image" src="img/cover11.jpg" />
```
Comment (realslimshanky) [inline index.html:229]: @YashKumarVerma, in your PR you have used `cover11.jpg` here as the background image for social embed block but I cannot find this image in the `/img/` directory and hence the background image failed to load. Could you please reply back with the image intended to be `cover11.jpg` as I'm fixing this issue as a part of #678.

Comment (realslimshanky) [reply index.html:229]: I've found the [image](https://github.com/fossasia/gci17.fossasia.org/blob/6d9a6ab4673b590d9901f4baf509e5ec8092463e/img/cover11.jpg) after looking at the repository at the point of time when you added the PR. I'll add that to the repository. Thanks anyway :)

**Pull request merged**